gilbreth_perception/package.xml

```diff
@@ -2,12 +2,12 @@
 <package>
   <name>gilbreth_perception</name>
   <version>0.0.0</version>
-  <description>The perception module</description>
+  <description>The gilbreth_perception package</description>
 
   <!-- One maintainer tag required, multiple allowed, one person per tag --> 
   <!-- Example:  -->
   <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
-  <maintainer email="jrgnichodeve@gmail.com">Jorge Nicho</maintainer>
```

Comment:
I'm still a maintainer :) . You can have multiple maintainers.

Reply:
Added:smile: